source/nims/index.md

```diff
@@ -0,0 +1,22 @@
+---
+review_priority: "index"
+html_theme.sidebar_secondary.remove: true
+---
+
+# NIMs Microservices
+
+`````{gridtoctree} 1 2 2 3
+:gutter: 2 2 2 2
+
+````{grid-item-card}
+:link: nemo
+:link-type: doc
```

Comment:
I'm undecided whether we should be lining to a page on our docs here, or whether we should just link directly to the NIM. The page will just be repeating the information from the NIM page, so maybe a direct link would be better. What do you think?

Reply:
Just tried the alternative link in my most recent commit - I think I like it better since it's definitely more straightforward - what about you?